alembic/versions/33de75e55858_adding_datetime_column.py

```diff
@@ -0,0 +1,44 @@
+"""Adding datetime column
+
+Revision ID: 33de75e55858
+Revises: 525b125ec799
+Create Date: 2015-10-10 12:32:39.936107
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '33de75e55858'
+down_revision = '525b125ec799'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('campaign_anon_view', sa.Column('datetime', sa.DateTime()))
+    op.create_index(op.f('ix_campaign_anon_view_datetime'), 'campaign_anon_view', ['datetime'], unique=False)
+    op.execute('UPDATE "campaign_anon_view" set datetime = created_at')
+    op.alter_column('campaign_anon_view', 'datetime', nullable=True)
```

Comment:
This should be `nullable=False`. Add explicit `nullable=True` above (True is default, but we provide it anyway to remove ambiguity).